needs rustup

It looks like rls build got broken by https://github.com/rust-lang/rust/commit/41316f0449025394fdca6606d3fdb3b8f37a9872 as well.

````
    Checking rls v1.35.0 (/home/matthias/vcs/github/rls)
warning: the item `Diagnostic` is imported redundantly
   --> rls/src/actions/post_build.rs:113:31
    |
15  | use crate::actions::diagnostics::{parse_diagnostics, Diagnostic, ParsedDiagnostics, Suggestion};
    |                                                      ---------- the item `Diagnostic` is already imported here
...
113 |         use crate::lsp_data::{Diagnostic, Position};
    |                               ^^^^^^^^^^
    |
    = note: #[warn(unused_imports)] on by default

error[E0061]: this function takes 5 parameters but 3 parameters were supplied
   --> rls/src/build/rustc.rs:265:12
    |
265 |         ls.register_late_pass(Some(sess), true, pass);
    |            ^^^^^^^^^^^^^^^^^^ expected 5 parameters

error: aborting due to previous error

For more information about this error, try `rustc --explain E0061`.
error: Could not compile `rls`.

````

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@ rls-vfs = "0.7"
 
 cargo = { git = "https://github.com/rust-lang/cargo", rev = "ed858daaf7c62f6f2156c47d00b0c669bcf9b183" }
 cargo_metadata = "0.7"
-clippy_lints = { git = "https://github.com/rust-lang/rust-clippy", rev = "92612c9de159889f77f05855a77482ee1d895f51", optional = true }
+clippy_lints = { git = "https://github.com/rust-lang/rust-clippy", rev = "25a11568fc5b995c2641560415e97244165dd278", optional = true }
 env_logger = "0.6"
 failure = "0.1.1"
 home = "0.3"

--- a/rls/src/actions/post_build.rs
+++ b/rls/src/actions/post_build.rs
@@ -110,7 +110,7 @@ impl PostBuildHandler {
         error: &failure::Error,
         stdout: &str,
     ) {
-        use crate::lsp_data::{Diagnostic, Position};
+        use crate::lsp_data::Position;
         use std::fmt::Write;
 
         // These notifications will include empty sets of errors for files

--- a/rls/src/build/rustc.rs
+++ b/rls/src/build/rustc.rs
@@ -262,7 +262,7 @@ fn clippy_after_parse_callback(compiler: &interface::Compiler) {
         ls.register_early_pass(Some(sess), true, false, pass);
     }
     for pass in late_lint_passes {
-        ls.register_late_pass(Some(sess), true, pass);
+        ls.register_late_pass(Some(sess), true, false, false, pass);
     }
 
     for (name, (to, deprecated_name)) in lint_groups {

